client-frameworks-support/client-support-ui5/rollup.config.js

```diff
@@ -0,0 +1,14 @@
+import copy from 'rollup-plugin-copy';
+export default {
+  input: 'src/ui5-support-lib.js',
```

Comment:
a simple copy? IMO, this can do more, you can even integrate the luigi client into the bundle if wanted then you just need to include the ui5-support-lib.js from here.

But we have two scenarios: 
  * include ui5-support-lib via shims
  * include ui5-support-lib via ui5-tooling-modules

For the first a all-in-one-bundle makes most sense, right? Or do you always want to consume the luigi-client independently?